src/api/index.ts

```diff
@@ -39,6 +39,23 @@ import * as converter from './converter';
 
 export * from './types';
 
+// setCookie sets new cookie key-value pair.
+export function setCookie(key: string, value: string) {
```

Comment:
The `api` folder contains functions related to the Yorkie API. 
(https://github.com/yorkie-team/dashboard/blob/main/design/file-structure.md)

How about moving the 'setCookie' and 'getCookie' functions to the [`utils`](https://github.com/yorkie-team/dashboard/tree/main/src/utils) folder?

Reply:
Is it necessary to set the `domain` and `path` for the cookie? We also use the cookie on the homepage. 
https://github.com/yorkie-team/yorkie-team.github.io/blob/25af19213edecc9a23cb9d83a1dc0aa1842398c1/components/Layout/Header.tsx#L15